README.md

```diff
@@ -1,74 +1,43 @@
 # MyParcel.com carrier API specification
 
-[![GitHub release](https://img.shields.io/github/release/MyParcelCOM/carrier-specification/all.svg)](https://github.com/MyParcelCOM/carrier-specification)
 [![Packagist](https://img.shields.io/packagist/v/MyParcelCOM/carrier-specification.svg)](https://packagist.org/packages/myparcelcom/carrier-specification)
 [![Swagger Validator](https://img.shields.io/swagger/valid/2.0/https/docs.myparcel.com/carrier-specification/swagger.json.svg)](https://online.swagger.io/validator/debug?url=https://docs.myparcel.com/carrier-specification/swagger.json)
 
 Description of the API specification used by MyParcel.com located at [https://docs.myparcel.com/carrier-specification](https://docs.myparcel.com/carrier-specification).
-This specification adheres to the [Swagger Specification](https://swagger.io/specification) v2.0.
+This specification adheres to the [Swagger 2.0 Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) and implements the [JSON API specification](http://jsonapi.org/).
 
 ## Content
 - [Installation](#installation)
-- [Validating the Spec](#validating-the-spec)
+- [Commands](#commands)
 - [Conventions](#conventions)
-- [Contributing](#contributing)
+- [License](#license)
 
 ## Installation
-The repository provides Docker containers to validate and preview the spec before committing changes.
-This is also used when validating pull requests. To setup the project for development run:
-```bash
-./mp.sh setup
-```
+The repository provides Docker containers to validate and preview the spec before committing changes. This is also used when validating pull requests. To install Docker, follow the steps in the [documentation](https://docs.myparcel.com/github/#docker).
 
-### Installing Docker
-To install Docker, follow the steps below for your preferred OS.
+To setup the project for development run:
 
-#### Mac
-Install Docker for Mac from [https://docs.docker.com/docker-for-mac/install](https://docs.docker.com/docker-for-mac/install).
-
-#### Windows
-Install Docker for Windows from [https://docs.docker.com/docker-for-windows/install](https://docs.docker.com/docker-for-windows/install).
-> **NOTE:** We recommend running the commands in a bash shell like Git Bash.
-
-#### Linux
-Install Docker by running the following command:
 ```bash
-curl -sSL https://get.docker.com/ | sh
+./mp.sh setup
 ```
 
-Then install Docker Compose by following the instructions [here](https://github.com/docker/compose/releases).
+## Commands
 
-Finally assign yourself to the Docker group:
-```bash
-sudo usermod -aG docker $(whoami)
-```
+`./mp.sh up` - Start the containers, it will start a server to watch file changes and reload automatically.
```

Comment:
Start the containers **which** will start a server to watch file changes and reload automatically.